java/data/pack.mcmeta.json

```diff
@@ -29,7 +29,7 @@
                     "description": "This is 1 for 1.13"
                 },
                 "description": {
-                    "type": "string",
+                    "$ref": "./data/pack.mcmeta.json#",
```

Comment:
```suggestion
                    "$ref": "../shared/text_component.json#",
```

Reply:
Oops, yes, that's what I meant to write. I'll fix it when I get to a PC.